pkg/rpc/server/server_test.go

```diff
@@ -455,10 +455,6 @@ var rpcTestCases = map[string][]rpcTestCase{
 				return &v
 			},
 		},
-		{
-			params: "1",
-			fail:   true,
```

Comment:
So why we're removing it?

Reply:
Because this test case is incorrectly constructed, it has malformed  `params` argument. With the current way of unmarshalling, `params`  must be a JSON array `[]`, and `"1"` can't be unmarshalled into array of `json.RawMessage`s, so this test is still failing but on earlier stage (response unmarshalling stage).